### PR TITLE
[PM-3334] Update desktop dark theme to match browser

### PR DESCRIPTION
**Pull request by danielleflinn**
## Type of change

```
- [ ] Bug fix
- [ ] New feature development
- [X] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

To simplify the color variables used in web clients, we should update the desktop app's dark theme to match that of the extension and web app. 

This will also allow the team to better transition the desktop client's UI to utilize the Component Library UI which is styled using the same variables as the web app and browser dark themes. See https://github.com/bitwarden/clients/pull/2241 for browser updates. 

## Code changes

- **clients/apps/desktop/src/scss/variables.scss** - updated variables based on the browser's dark theme, removed the unused `groupingsActiveColor` variable
- **clients/apps/desktop/src/scss/plugins.scss** - updated toast text and icon button to be compliant with WCAG contrast criteria 

## Screenshots
Screenshot 2023-07-25 at 1 26 34 PM
Screenshot 2023-07-25 at 1 26 54 PM
Screenshot 2023-07-25 at 1 27 05 PM
Screenshot 2023-07-25 at 1 27 20 PM

## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team
- Ensure that all UI additions follow [WCAG AA requirements](https://contributing.bitwarden.com/contributing/accessibility/)

### CODE REVIEW

**Comment by willmartian** [comment]
Here is what the CL dialogs (from https://github.com/bitwarden/clients/pull/5875) look like with the theme changes in this PR:

https://github.com/bitwarden/clients/assets/17113462/1305926c-3762-4a78-b249-ca485c1a6f78

File: apps/desktop/src/main/window.main.ts [hunk 1]
```diff
@@ -236,7 +236,7 @@ export class WindowMain {
       case "light":
         return "#ededed";
       case "dark":
-        return "#222222";
+        return "#2f343d";
```
Comment (willmartian) [inline apps/desktop/src/main/window.main.ts:239]: Before, this value matched`backgroundColorAlt2`. It now matches `backgroundColorAlt`. 

Was this intentional?

**Review by willmartian — COMMENTED** [review]
Nice work! One clarifying question:

File: apps/desktop/src/main/window.main.ts (hunk 1 above)
Comment (danielleflinn) [reply apps/desktop/src/main/window.main.ts:239]: Good catch, no that was not intentional. I'll update

**Review by willmartian — APPROVED** [review]

**Review by Hinton — APPROVED** [review]

**Pull request merged**